api/configuration/index.ts

```diff
@@ -42,6 +42,8 @@ export const getEnvironment = () => process.env.NODE_CONFIG_ENV
  */
 export const getConfigValue = reference => config.get(reference)
 
+export const hasConfigValue = reference => config.has(reference)
```

Comment:
check with @ficklephil but I thought he said something about the `config.has()` doesn't quite do what we expect?